SoftLayer/CLI/virt/detail.py

```diff
@@ -69,11 +69,13 @@ def cli(env, identifier, passwords=False, price=False):
     table.add_row(['transient', result.get('transientGuestFlag', False)])
     table.add_row(['created', result['createDate']])
     table.add_row(['modified', result['modifyDate']])
-    last_transaction = "{} ({})".format(utils.lookup(result, 'lastTransaction', 'transactionGroup', 'name'),
-                                        utils.clean_time(utils.lookup(result, 'lastTransaction', 'modifyDate')))
+    last_transaction = ''
+    if result.get('lastTransaction') != []:
```

Comment:
You can just do the following, which will also work if `lastTransaction` is None.  Empty list and dictionaries return `False` in python.

```suggestion
    if result.get('lastTransaction'):
```